[{"username": "aspinuso", "errors": " Compute Error: Traceback (most recent call last):\n  File \"/Users/aspinuso2/Documents/workspace/verce-hpc-pe-prod/src/verce/processing.py\", line 545, in __computewrapper\n    self.compute()\n  File \"/Users/aspinuso2/Documents/workspace/verce-hpc-pe-prod/src/kmlGenerator_INGV.py\", line 152, in compute\n    f=open(stationsfile,'r')\nIOError: [Errno 2] No such file or directory: u'../test-resources/specfem3dseismogram//../../DATA/STATIONS_FILTERED'\n\n", "derivationIds": [], "parameters": [{"val": "../test-resources/specfem3dseismogram//../../DATA/STATIONS_FILTERED", "key": "stations_filtered"}, {"val": "../OUTPUT_FILES/TRANSFORMED/PLOT/", "key": "images_path"}, {"val": "../test-resources/specfem3dseismogram//../../DATA/Par_file", "key": "par_file"}, {"val": "../test-resources/specfem3dseismogram//../../DATA/CMTSOLUTION", "key": "cmt_solution"}], "instanceId": "Invoker-instance-orfeus-as", "iterationIndex": null, "pid": "10122", "site": "", "stateful": false, "endTime": "2014-07-18 02:47:08.115374", "streams": [], "runId": "bettercleaning01391768324782", "startTime": "2014-07-18 02:47:08.114921", "_id": "kmlGenerator_INGVorfeus-as-10122-cf38e25e-0e25-11e4-b33f-10ddb1b266f3", "type": "lineage", "annotations": [], "name": "kmlGenerator_INGV"}]